### ISSUE TITLE
Why my debug image output is all black?

### ISSUE THREAD

**Issue by Gengzigang** [meteorshowers/RCF-pytorch#27]
![sss](https://user-images.githubusercontent.com/37038881/63007743-cf6d9700-beb3-11e9-95b6-a1d49000e153.png)
Is it normal?
The last three output maps are always all black after many epoches.

**Comment by LIMENG0307** [comment]
I met the same situation,did you know it。

**Comment by Gengzigang** [comment]
> I met the same situation,did you know it。

Maybe the learning rate is too high.

**Comment by khflam** [comment]
> > I met the same situation,did you know it。
> 
> Maybe the learning rate is too high.

Hi Gengzigang, did you manage to fix the black images issue? Just curious whether the learning rate was the cause.

**Comment by LunarSun** [comment]
> ![sss](https://user-images.githubusercontent.com/37038881/63007743-cf6d9700-beb3-11e9-95b6-a1d49000e153.png)
> Is it normal?
> The last three output maps are always all black after many epoches.

I met the same problem,Have you solved it?

**Comment by zcong17huang** [comment]
> ![sss](https://user-images.githubusercontent.com/37038881/63007743-cf6d9700-beb3-11e9-95b6-a1d49000e153.png)
> Is it normal?
> The last three output maps are always all black after many epoches.
 I met the same problem as you, Have you solved it? Please tell me.

**Comment by zcong17huang** [comment]
> ![sss](https://user-images.githubusercontent.com/37038881/63007743-cf6d9700-beb3-11e9-95b6-a1d49000e153.png)
> Is it normal?
> The last three output maps are always all black after many epoches.

I met the same problem as you, Have you solved it? Please tell me.

**Comment by zcong17huang** [comment]
I think i have found the solution. Change the kernel size of [upsample layer5](https://github.com/meteorshowers/RCF-pytorch/blob/master/models.py#L141) from 32 to 16, the problem will be solved.
